.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# Contributing to Linux Guest Agent
+First, thank you for contributing to Azure Functions Python Worker repository!
+
+## Basics
+If you would like to become an active contributor to this project, please follow the instructions provided in [Microsoft Azure Projects Contribution Guidelines](http://azure.github.io/guidelines/).
+
+## Table of Contents
+[Before starting](#before-starting)
+- [Github basics](#github-basics)
+- [Code of Conduct](#code-of-conduct)
+
+[Making Changes](#making-changes)
+- [Pull Requests](#pull-requests)
+- [Pull Request Guidelines](#pull-request-guidelines)
+    - [Cleaning up commits](#cleaning-up-commits)
+    - [General guidelines](#general-guidelines)
+    - [Testing guidelines](#testing-guidelines)
+
+## Before starting
+
+### Github basics
+
+#### GitHub workflow
+
+If you don't have experience with Git and Github, some of the terminology and process can be confusing. [Here's a guide to understanding Github](https://guides.github.com/introduction/flow/).
+
+#### Forking the Azure/Guest-Configuration-Extension repository
+
+Unless you are working with multiple contributors on the same file, we ask that you fork the repository and submit your Pull Request from there. [Here's a guide to forks in Github](https://guides.github.com/activities/forking/).
```

Comment:
We should mention that we **block** PRs from forks to go through E2E tests. However, we will cherry-picked PRs once they pass unit tests.